### DISCUSSION TITLE
More descriptive env variable for GitHub API token

### DISCUSSION THREAD

**Discussion by roadkell** [marcosnils/bin#174] [Ideas]
For example, how about `BIN_GITHUB_API_TOKEN`, similarly to the one used by [Homebrew](https://docs.brew.sh/Manpage)?

Also, thanks for the awesome project!

**Comment by marcosnils** [comment]
this decision was intentional so it works out of the box with other tools that use `GITHUB_AUTH_TOKEN` like gorelease. I'd prefer not to change it since it's a backwards incompatible change :pray:

**Reply by roadkell** [reply]
I see, that's a neat solution indeed. Thanks.
